README.md

```
@@ -94,6 +102,26 @@ on each MySQL server instance run the `proxysql_binlog_reader`, e.g:
 ./proxysql_binlog_reader -h 127.0.0.1 -u root -p rootpass -P 3306 -l 6020 -f
 ```
 
+#### TLS connections
+
+TLS is required and server-certificate verification is enabled by default. For a
+verified connection, provide the CA that issued the MySQL server certificate:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)